notebooks/text_generator.livemd

```diff
@@ -0,0 +1,920 @@
+```
+
+Lowercase English characters in ASCII encoding are presented with numbers between `97 = a` and `122 = z` a space is `32 = [ ]` a new line is `10 = \n` and the period is `46 = .`.
```

Comment:
More precisely, the charlist includes unicode code points, it's just that in our case it matches ASCII.

Reply:
Addressed in 7a502c4